tasks/main.yml

```diff
@@ -78,6 +79,7 @@
     name: docker
     state: started
     enabled: true
+  when: not ansible_check_mode
```

Comment:
why are you using `not ansible_check_mode` here?

the [ansible docs](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/meta_module.html#attribute-check_mode) say: 

> While these actions don’t modify the targets directly they do change possible states of the target within the run